oot.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/bash
+
+
+function usage {
+	echo 'Usage:'
+	echo -e "\t$0 <device>\t\t(Example for akatsuki: $0 akatsuki)"
+	echo -e "\t$0 -c gcc <device>\t(Will build with GCC toolchain)"
+	echo -e "\t$0 -f <device>\t\t(Will flash the kernel using fastboot)"
+	exit 1
+}
+
+
+while getopts ':c:f' OPT; do
+	case ${OPT} in
+	c)
+		case ${OPTARG} in
+		clang|gcc|linaro_gcc)
+			export _compiler=${OPTARG}
+			;;
+		*)
+			echo "Compiler '${OPTARG}' unknown or not implemented"
+			exit 1
+			;;
+		esac
```

Comment:
(Following https://github.com/MarijnS95/oot/pull/1/files#r497809052)

```suggestion
		if [ -f "$_self_dir/compile_$OPTARG.sh" ]; then
			_compiler=${OPTARG}
		else
			echo "Compiler '${OPTARG}' unknown or not implemented"
			exit 1
		fi
```
Assuming you move the assignment of `_self_dir` on line 47 above line 13.

Reply:
OK. I was just writing what I didn't understand about it and then it just popped in my head what you meant :)

Reply:
Hehe a ~picture~ code example says more than a thousand words :+1:

Reply:
Done.